### ISSUE TITLE
PAYLOAD=network_only.txt overwrites setup.cfg

### ISSUE THREAD

**Issue by straikongit** [mame82/P4wnP1#276]
When choosing the payload "PAYLOAD=network_only.txt" the settings for autossh in setup.config have no effect because they get overwritten by the settings in network_only.txt.
Sinply commenting out the settings in network_only.txt fix this issue.

**Comment by Swiftb0y** [comment]
Yes that is generally the case. Please refer to the Docs when you have further issues. 
https://p4wnp1.readthedocs.io/en/latest/Troubleshooting/#my-settings-in-setupcfg-are-not-used-by-the-payload
